### PR TITLE
Rc/case search accessibility

### PR DESCRIPTION
**Pull request by Robert-Costello**
## Product Description

Previously, in case search, there was no indication that screenreader users had landed on the case search page. Search field labels were skipped if the user was tabbing, as opposed to stepping incrementally, through the page, and text input fields were read out as generically as "text entry".

Now the case search heading and search field labels will be read when tabbing and the text inputs will be read with a more specific description, e.g. "last name text entry".

https://user-images.githubusercontent.com/42388648/166567392-bf789716-f763-4e20-b4de-7d0fde190076.mov

## Technical Summary

https://dimagi-dev.atlassian.net/browse/USH-1929

### Safety story

Tested locally and on staging.

### QA Plan

no QA

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/cloudcare/templates/formplayer/query_view.html [hunk 1]
```diff
@@ -53,14 +53,14 @@
     <select>
 
     <% } else if (input == "daterange") { %>
-    <input type="text" class="daterange query-field form-control" value="<%- value %>">
+    <input aria-label="<%- text ? text : "" %> daterange" type="text" class="daterange query-field form-control" value="<%- value %>">
```
Comment (AddisonDunn) [inline corehq/apps/cloudcare/templates/formplayer/query_view.html:56]: Does the screenreader know how to read "daterange" (without a space)?

**Review by AddisonDunn — APPROVED** [review]
Nice. It looks like "after submitting a search, CommCare does not provide feedback to the user" is part of the ticket too, do we know what would that look like to implement?

**Comment by Robert-Costello** [comment]
> It looks like "after submitting a search, CommCare does not provide feedback to the user" is part of the ticket too
I took this as part of the broader research into what needs to be looked at and the specific points about text fields etc as the actionable points. I think we'd need to get more specific guidance on what user feedback would be helpful before knowing what would the possible changes would be.

File: corehq/apps/cloudcare/templates/formplayer/query_view.html (hunk 1 above)
Comment (Robert-Costello) [reply corehq/apps/cloudcare/templates/formplayer/query_view.html:56]: You mean "daterange" vs "date range", correct? That's an interesting question. It does read it correctly, but if I hadn't already tested already I would wonder the same thing.

Comment (AddisonDunn) [reply corehq/apps/cloudcare/templates/formplayer/query_view.html:56]: Yup exactly and it works anyways so 👍

**Comment by AddisonDunn** [comment]
> I think I have a solution for this, actually. Adding a change to make the results page heading reachable with an aria label.

Nice.

**Pull request merged**